change/@microsoft-fast-tooling-f62e9316-be2f-4e57-a890-1ef2c4bf54aa.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "integrate html renderer into creator",
```

Technically this is the change for this package.

```suggestion
  "comment": "add checks in the HTML render component for window scroll and resize events",
```